pyproject.toml

```
@@ -15,3 +15,10 @@ lint = [
     "pylint",
 ]
 log_level = "INFO"
+
+[tool.coverage.run]
+omit = [
+    "*/__init__.py",
+    "*/tests/*",
+    "*/migrations/*",
```

Pourrais-tu ajouter les imports Galion & Ecolo à la liste des exclusions ?

```suggestion
    "*/migrations/*",
    "ecoloweb/*",
    "programmes/management/commands/import_galion.py",
```